docs/walkthrough/conditional-transitions.md

```diff
@@ -20,33 +20,29 @@ states:
 - id: ifelse
   type: switch
   conditions:
-  - condition: '.names'
+  - condition: jq(.names)
     transition: poster
 - id: poster
   type: action
   action:
     function: httprequest
-    input: '{
-      "method": "POST",
-      "url": "https://jsonplaceholder.typicode.com/posts",
-      "body": {
-        "name": .names[0]
-      }
-    }'
-  transform: 'del(.names[0])'
+    input:
+      method: "POST"
+      url: "https://jsonplaceholder.typicode.com/posts"
+      body:
+        name: jq(.names[0])
+  transform: jq(del(.names[0]))
   transition: ifelse
 ```
 
 ### Input
```

Comment:
This section was intended to show what the input to the action would be. I'd prefer it stayed JSON.